schema.sql

```diff
@@ -1,10 +1,12 @@
 /* Database schema to keep the structure of entire database. */
-CREATE SCHEMA `vet_clinic` ;
-CREATE TABLE `vet_clinic`.`animals` (
-  `idAnimals` INT NOT NULL AUTO_INCREMENT,
-  `name` VARCHAR(45) NOT NULL,
-  `date_of_birth` DATETIME NOT NULL,
-  `escape_attempts` INT NOT NULL,
-  `neutered` TINYINT(1) NOT NULL,
-  `weight_kg` DECIMAL(3) NOT NULL,
-  PRIMARY KEY (`idAnimals`));
\ No newline at end of file
+CREATE DATABASE vetclinic
+CREATE TABLE public.animals
```

Comment:
- [ ] As stated by the previous reviewer, please use the syntax `CREATE TABLE animals`  by removing the `public `keyword, which is not necessary.  Kindly just use:

```SQL
CREATE TABLE animals
```